index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SMACSS Demo</title>
+    <link href="css/reset.css" rel="stylesheet"/>
+    <link href="css/base.css" rel="stylesheet"/>
+    <link href="css/layout.css" rel="stylesheet"/>
+    <link href="css/module.css" rel="stylesheet"/>
+  </head>
+  <body>
+    <header>a</header>
+    <aside>b</aside>
+    <main>
+      <section>
+        <figure class="cd">c</figure>
+        <figure class="cd">d</figure>
+      </section>
+      <section>
```

Comment:
Similar to notes for the previous section element. Instead of giving every child the same class, leverage your well-structured HTML page.